marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/flows/suite-setup.xqy

```diff
@@ -0,0 +1,44 @@
+xquery version "1.0-ml";
```

Comment:
This setup seems like it'd be very useful for any test that wants to easily setup and run a flow with mapping and mastering. No need to move it yet, but I'm guessing it would be useful to have this outside of the suites directory and in an e.g. "/test/com.marklogic.hub/" directory full of DHF-specific helpers.